docs/tutorials/scenario-simulation/index.md

```diff
@@ -0,0 +1,5 @@
+# Scenario simulation
+
+!!! warning
+
+    Under Construction
```

Comment:
Don't you need to update the `.pages` file?

https://github.com/autowarefoundation/autoware-documentation/blob/62b96a9e0f8cb50d9de197550263dbce59647602/docs/tutorials/scenario-simulation/.pages

Reply:
Thank you :+1:

Reply:
[4836e3d](https://github.com/autowarefoundation/autoware-documentation/pull/358/commits/4836e3d0c058f02e734f7a7a06a2ed2708393858)